test/recipe/Baked.Test.Recipe.Service.Application/Program.cs

```diff
@@ -27,6 +28,8 @@
             claims: ["User", "Admin", "BaseA", "BaseB", "GivenA", "GivenB", "GivenC"],
             baseClaims: ["BaseA", "BaseB"]
         ),
+        core: c => c.Dotnet(baseNamespace: _ => "Baked.Test")
+            .ForNfr(c.Dotnet(entryAssembly: Assembly.GetExecutingAssembly(), baseNamespace: _ => "Baked.Test")),
```

Comment:
this is not a good solution. but i'll add this as an issue where we'll find a way to catch the correct assembly automatically.